src/App.scss

```diff
@@ -535,9 +554,9 @@ model-viewer {
         margin-top: 10%;
         margin-bottom: 0%;
 
-        &__text {
-            font-size: 18px;
-            margin-left: -22%;  
+        .home__robot__text {
+            margin-left: -22%;
+            font-size: 15px;  
```

Comment:
@sarahraisuddin Instead of using .home__robot__text, replace with &__text. Please fix that there and anywhere else in the code. I also see it on line 365 and 370.